src/app/clusters/scenes-server/scenes-server.h

```diff
@@ -34,6 +34,28 @@ namespace Scenes {
 class ScenesServer : public CommandHandlerInterface, public AttributeAccessInterface
 {
 public:
+    static constexpr size_t kScenesServerMaxEndpointCount =
+        EMBER_AF_SCENES_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT;
+    static_assert(kScenesServerMaxEndpointCount <= kEmberInvalidEndpointIndex, "Scenes endpoint count error");
+    static constexpr uint8_t kScenesServerMaxFabricCount = CHIP_CONFIG_MAX_FABRICS;
+
+    // FabricSceneInfo
+    class FabricSceneInfo
+    {
+    public:
+        Span<Structs::SceneInfoStruct::Type> GetFabricSceneInfo(EndpointId endpoint);
+        Structs::SceneInfoStruct::Type * GetSceneInfoStruct(EndpointId endpoint, FabricIndex fabric);
+        CHIP_ERROR SetSceneInfoStruct(EndpointId endpoint, FabricIndex fabric, Structs::SceneInfoStruct::Type & sceneInfoStruct);
+        void ClearSceneInfoStruct(EndpointId endpoint, FabricIndex fabric);
+
+    private:
+        CHIP_ERROR FindFabricSceneInfoIndex(EndpointId endpoint, size_t & endpointIndex);
+        CHIP_ERROR FindSceneInfoStructIndex(FabricIndex fabric, size_t endpointIndex, uint8_t & index);
```

Comment:
So this index is safe to be uint8_t because it's no more than CHIP_CONFIG_MAX_FABRICS?  Would be good to have that documented.

Reply:
Added to the parameter description and copied in the .h file